### PR TITLE
[miaopai] Add new extractor

### PR DESCRIPTION
**Pull request by xyb**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/rg3/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/rg3/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests

### What is the purpose of your *pull request*?
- [ ] Bug fix
- [x] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

Explanation of your *pull request* in arbitrary form goes here. Please make sure the description explains the purpose and effect of your *pull request* and is worded well enough to be understood. Provide as much context and examples as possible.

This is an extractor for miaopai (miaopai.com), which is a Chinese video app lets users shoot, edit, and share 10 second videos.

### CODE REVIEW

File: youtube_dl/extractor/miaopai.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        request.add_header('User-Agent', self._USER_AGENT_IPAD)
+        webpage = self._download_webpage(request, video_id)
+
+        title = self._html_search_regex(r'<title>([^<]*)</title>',
+                                        webpage,
+                                        'title')
+        regex = r"""<div *class=['"]video_img[^>]*data-url=['"]([^'"]*\.jpg)['"]"""
+        thumbnail = self._html_search_regex(regex, webpage, '')
+        regex = r"""<video *[^>]*src=['"]([^'"]*)['"]""",
+        video_url = self._html_search_regex(regex, webpage, '')
```
Comment (yan12125) [inline youtube_dl/extractor/miaopai.py]: Use ```self._parse_html5_media_entries``` instead.

Comment (xyb) [inline youtube_dl/extractor/miaopai.py]: @yan12125 updated.

**Comment by yan12125** [comment]
Thanks merged. Would you like to be listed in [AUHTORS](https://github.com/rg3/youtube-dl/blob/master/AUTHORS)? If so under what name?

**Comment by xyb** [comment]
@yan12125 Sure, it's my honour, my name is "Xie Yanbo".

**Comment by yan12125** [comment]
Done. Thanks for contribution :)

**Pull request merged**